packages/github/src/github.ts

```diff
@@ -14,7 +14,12 @@ export const context = new Context.Context()
  */
 export function getOctokit(
   token: string,
-  options?: OctokitOptions
+  options?: OctokitOptions,
+  ...additionalPlugins: OctokitPlugin[]
 ): InstanceType<typeof GitHub> {
+  if (additionalPlugins.length) {
```

Comment:
Depending on how the args are ultimately structured, may be worth adding a null check here via optional chaining, in case additionalPlugins is left out entirely?  (to prevent "cannot read property length of undefined" type errors?).

```suggestion
  if (additionalPlugins?.length) {
```